docs/standard-library/built-in-decorators.md

```diff
@@ -116,11 +116,11 @@ Specify how to encode the target type.
 | encodedAs | `Scalar` | What target type is this being encoded as. Default to string. |
 
 #### Examples
-##### offsetDateTime encoded with rfc7231
+##### offsetDateTime encoded with http-date
 
 
 ```tsp
-@encode("rfc7231")
+@encode("http-date")
```

Comment:
Oh I wouldn't change the known encoding to be `http-date` I think just the mapping from `rfc7231-> http-date` for openapi3.

All the emitters already depend on knowing `rfc7231` as a datetime encoding

Reply:
Ah ... okay ... I will fix.

Reply:
I think you forgot this example

Reply:
I left this example as "http-date" since I think that is more helpful to external users. It is just an example ... it is not describing the behavior of the possible or even likely values. Does it really need to be "rfc7231" in the public docs?

Reply:
well as we have established it as one of the "known" encoding in the enum this feels like the way to say I want a rfc7231 date in typespec should be like that. It just happens to be that openapi3 the convention is to have a format called `http-date`.

I don't think we should show 2 different ways to do rfc7231 nor recommend renaming one of the known encoding  just to satisfy an emitter. Feel like its the job of the emitter to do its best with our documented behavior

Reply:
okay ... reverted this change.